src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.Fields.Edit.cshtml

```diff
@@ -0,0 +1,54 @@
+@using OrchardCore.Forms.ViewModels;
+
+@model ValidationRulePartEditViewModel
+@{
+    dynamic dyModel = Model;
```

Comment:
There must be a better way to do it. Model is probably a shape here, hence `IShape` which has `Metadata`.